mira/dkg/api.py

```diff
@@ -328,6 +334,95 @@ def get_relations(
         return [RelationResponse(subject=s, predicate=p, object=o) for s, p, o in records]
 
 
+@api_blueprint.post(
+    "/relations_graph",
+    response_model=None,
+    tags=["relations"],
+)
+def get_relations_graph(
+    request: Request,
+    background_tasks: BackgroundTasks,
+    relation_query: RelationQuery = Body(
+        ...,
+    )
```

Comment:
This is not exactly what I had in mind, we don't necessarily need a REST API (i.e., server side) endpoint that returns a graph visualization. Rather, we would want a client side visualization function which takes as input a data structure of the form the `relations` endpoint returns, and runs the code below.

Reply:
I reimplemented everything to solve this.